mongo/create.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-undef */
+if (db.getCollectionNames().indexOf('decisionDatabase') >= 0) {
```

Comment:
It would be good to import `db` explicitly instead of pulling it from the global namespace, do you know where it is coming from?